modules/rust-wasm-test/src/lib.rs

```diff
@@ -52,6 +52,15 @@ pub struct Point {
     y: i64,
 }
 
+// Test we can compile multiple constraints
+#[spacetimedb(table)]
```

Comment:
This is added to check it compiles.

Reply:
This looks good. You might have to change an integration smoke test which reads specific log lines because another table will add a log line and through the numbers off by one.